examples/Reranking_search_results_with_cross-encoders.ipynb

```diff
@@ -0,0 +1,805 @@
+    "Relevant to the query?\n",
+    "Assistant: No\n",
+    "\"\"\"\n",
+    "End of Examples\n",
+    "\n",
+    "Begin!\n",
+    "\n",
+    "Query: {query}\n",
+    "Retrieved document: {document}\n",
+    "Relevant to the query?\n",
+    "Assistant:\n",
+    "'''\n",
+    "\n",
+    "\n",
+    "@retry(wait=wait_random_exponential(min=1, max=40), stop=stop_after_attempt(3))\n",
+    "def rerank_document(query, document, output_list):\n",
+    "    response = openai.Completion.create(\n",
+    "        model=\"text-davinci-003\",\n",
+    "        prompt=prompt.format(query=query, document=content),\n",
+    "        temperature=0,\n",
+    "        logprobs=1,\n",
+    "        logit_bias={\n",
+    "            5297: 1,\n",
+    "            2949: 1\n",
+    "        }\n",
+    "    )\n",
+    "\n",
+    "    output_list.append(\n",
+    "        (\n",
+    "            query,\n",
+    "            document,\n",
+    "            response[\"choices\"][0][\"text\"],\n",
+    "            response[\"choices\"][0][\"logprobs\"][\"token_logprobs\"][0],\n",
+    "        )\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "753cd363",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "content = result_list[0][\"title\"] + \": \" + result_list[0][\"summary\"]\n",
+    "\n",
+    "# Set logprobs to 1 so we return the most likely token with the likelihood\n",
```

Comment:
nit: maybe rephrase this comment? wasn't sure what it was saying when I first read it

Reply:
Updated wording here, it was confusing